### PR TITLE
Feature/static from address

### PR DESCRIPTION
**Pull request by yanokwa**
Not sure you want this feature, but figured I'd contribute it anyway. I prefer having a static return address.

### CODE REVIEW

File: index.js [hunk 1]
```diff
@@ -1,16 +1,18 @@
...
 // Expected keys/values:
+// - fromEmail: Emails will appear to come from this verified address
 // - emailBucket: S3 bucket name where SES stores emails.
 // - emailKeyPrefix: S3 key name prefix where SES stores email. Include the
 //   trailing slash.
 // - forwardMapping: Object where the key is the email address from which to
 //   forward and the value is an array of email addresses to which to send the
 //   message.
 var defaultConfig = {
+  fromEmail: "noreply@example.com"
```
Comment (arithmetric) [inline index.js]: This line needs a comma at the end. It's causing an error in the tests.

**Comment by arithmetric** [comment]
@yanokwa Thanks for this contribution. I commented on a syntax error that is causing an error in the tests. Could you resolve that in your branch?

**Comment by yanokwa** [comment]
Well, the good news is that I've fixed the linting errors. The bad news is that not sure how the testing framework works (I'm new to npm) so not confident trying to write the tests.

**Comment by arithmetric** [comment]
Thanks! This is merged and released as v2.3.0.

**Pull request merged**